apps/dashboard/src/pages/usecase-select-page.tsx

```diff
@@ -41,6 +41,14 @@ export function UsecaseSelectPage() {
   const [selectedUseCases, setSelectedUseCases] = useState<ChannelTypeEnum[]>([]);
   const [hoveredUseCase, setHoveredUseCase] = useState<ChannelTypeEnum | null>(null);
 
+  useEffect(() => {
```

Comment:
We can leverage browser [resource hints prefetching](https://web.dev/learn/performance/resource-hints) and let then decide when is the best time to prefetch them:

```import React from "react";
import { Helmet } from "react-helmet-async";

const ImagePrefetcher = () => {
  return (
    <Helmet>
      <link rel="prefetch" href="/path/to/image1.jpg" as="image" />
      <link rel="prefetch" href="/path/to/image2.jpg" as="image" />
    </Helmet>
  );
};
```